spec/requests/api/collections_spec.rb

```diff
@@ -119,6 +119,11 @@ def test_collection_bulk_query(collection, collection_url, klass, id = nil)
       test_collection_query(:flavors, flavors_url, Flavor)
     end
 
+    it "query FloatingIps" do
```

Comment:
@imtayadeway, sure although I believe all those test could go in the collection but that would be the purpose of another PR ;)